index.js

```diff
@@ -68,7 +88,17 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies,id) {
+ 
+  if(movies.length===0){
+    throw error
+  }
+  return movieID = movies.find(
+    (movie)=> movie.imdbID === id || null
+  );
+ 
+  // return movieID
```

Comment:
You were **really**  close on this one! 
```suggestion
  let movieID = movies.find(
    (movie)=> movie.imdbID === id
  );
 
   return movieID || null
```